extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/exporter/OtlpExporterTracesConfig.java

```diff
@@ -89,9 +91,9 @@ interface TrustCert {
         Optional<List<String>> certs();
     }
 
-    public static class Protocol {
+    class Protocol {
         public static final String GRPC = "grpc";
-        public static final String HTTP_PROTOBUF = "http/protobuf";
-        public static final String HTTP_JSON = "http/json";
+        public static final String HTTP_PROTOBUF = "http";
```

Comment:
The correct value is "http/protobuf", per spec.
See: https://github.com/open-telemetry/opentelemetry-java/blob/7ae32c90e012d87000bd3f64b9c4b3b937f85de9/exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/internal/OtlpConfigUtil.java#L42

Reply:
I was following the suggestion by @radcortez [here](https://github.com/quarkusio/quarkus/issues/21535#issuecomment-1658218695). Let me know which one you guys want, and I'll update the PR

Reply:
The spec doesn't define the specific string to use, as far as I can see. That's an implementation prerogative and they chose the values from the link above.

Reply:
In any case, I updated the PR

Reply:
It's fine, use `http/protobuf`.